### ISSUE TITLE
Floating point exception(core dumped)

### ISSUE THREAD

**Issue by fkxyyds** [Pointcept/SAMPart3D#16]
After executing sh scripts/train.sh -g 1 -d sampart3d -c sampart3d-trainmlp-render16views -n knight -o knight，the following message always appears.

[2025-01-16 07:38:40,907 INFO train.py line 166 1886571] >>>>>>>>>>>>>>>> Start Training >>>>>>>>>>>>>>>>
[2025-01-16 07:38:40,907 INFO train.py line 171 1886571] => Loading checkpoint & weight ...
/home/guest/fankangxu/SAMPart3D/SAMPart3D-main/exp/sampart3d/knight/code/pointcept/engines/train.py:173: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  checkpoint = torch.load(
[2025-01-16 07:38:41,108 INFO train.py line 192 1886571] Missing keys: ['scale_statistics', 'instance_net.params', 'pos_net.params']
/home/guest/anaconda3/envs/sampart3d/lib/python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 3 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
scripts/train.sh: line 101: 1886571 Floating point exception(core dumped) $PYTHON "$CODE_DIR"/launch/$TRAIN_CODE --config-file "$CONFIG_DIR" --num-gpus "$GPU" --options save_path="$EXP_DIR" oid="$OID" label="$LABEL"

**Comment by EricWang12** [comment]
I encountered the same problem 🧐

**Comment by fkxyyds** [comment]
> I encountered the same problem 🧐

So have you already resolved it?

**Comment by otanaaa** [comment]
Same promblem. Have anyone solved it?

**Comment by yhyang-myron** [comment]
It may also be due to a version issue with the torch-scatter library.
This issue may be resolved through wheel installation of the torch-scatter library.

**Comment by fkxyyds** [comment]
> It seems to be related to the version of PyTorch. I am using CUDA 12.4, and the error occurred with `torch==2.5.1`, but I was able to successfully run it with `torch==2.4.0`.
I had solved it with your method.Thanks!
